raml-parser-2/src/test/resources/org/raml/v2/parser/resource-types/global-media-type-merge/merge-different-media-types/input.raml

```diff
@@ -0,0 +1,15 @@
+#%RAML 1.0
+
+title: API Manager
+
+resourceTypes:
+  collection:
+    post:
+      body:
+        application/json:
+
+/tiers:
+  type: collection
+  post:
+    body:
+      text/xml:
```

Comment:
Based on name, this test is checking the merging when global media type is present. But you didn't define a global one

Reply:
done